### DISCUSSION TITLE
Alpha 173

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#5831] [Release Notes]
- Added a hotkey to split all panes into separate tabs
- Added a hotkey to combine all tabs into the current one as panes
- Added an optional profile icon to tabs (Martin Grulich) [#5767](https://github.com/Eugeny/tabby/pull/5767) - enable it in Settings -> Window.
- macOS: fixed app not quitting completely on Cmd+Q (Michael Wizner) [#5743](https://github.com/Eugeny/tabby/pull/5743)
- Locale: added Bulgarian
- SSH: HTTP proxy support
- SSH: try other OpenSSH key types besides RSA from `~/.ssh`
- Bundled a Powerline-enabled fallback font - now you can use a non-Powerline font with a Powerline prompt and icons.
- Fixed docked window overlapping the Windows taskbar
- Show recent profiles in jump lists & dock menu
- Passthrough arrows+modifiers when in alternate buffer mode

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.173'>Alpha 173</a>.</em>

**Comment by SynapseScribe** [comment]
Thank you so much for fixing the taskbar overlap. Now it looks perfect !
